partcad/src/partcad/converter.py

```
@@ -0,0 +1,169 @@
+            raise ValueError("Failed to read the STEP file")
+        reader.TransferRoots()
+        return reader.OneShape()
+
+    @staticmethod
+    def _write_step(shape: TopoDS_Shape, step_file: str):
+        """Write a shape to a STEP file."""
+        writer = STEPControl_Writer()
+        mode = STEPControl_AsIs  # Use STEPControl_AsIs for default mode
+        writer.Transfer(shape, mode)
+        if writer.Write(step_file) != 1:
+            raise IOError(f"Failed to write STEP file: {step_file}")
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)